### ISSUE TITLE
Nothing RPROVIDES 'nativesdk-catkin'

### ISSUE THREAD

**Issue by miram-jraidi** [ros/meta-ros#856]
i am trying to build a Linux image with meta-ros-dashing  layer branch dunfell but when I tried to bitbake ros-image-sdktest
this error always shows up 
Loading cache: 100% |############################################| Time: 0:00:00
Loaded 4335 entries from dependency cache.
NOTE: Resolving any missing task queue dependencies
ERROR: Nothing RPROVIDES 'nativesdk-catkin'
NOTE: Runtime target 'nativesdk-catkin' is unbuildable, removing...
Missing or unbuildable dependency chain was: ['nativesdk-catkin']
ERROR: Required build target 'ros-image-sdktest' has no buildable providers.
Missing or unbuildable dependency chain was: ['ros-image-sdktest', 'nativesdk-catkin']

**Comment by hannes09** [comment]
Same error comes also with sdktest of ros-noetic.

I have addded `BCLASSEXTEND=“native nativesdk” ` to a catkin_%.bbappend

That will build the sdk.

To use th sdk I  sourced the setup.bash of the sdk like this:

`source my_sdk/x86_64-tdsdk-linux/opt/ros/noetic/setup.bash`

But unfortunataly I got:

`CMake Error at CMakeLists.txt:65 (message):
  find_package(catkin) failed.  catkin was neither found in the workspace nor
  in the CMAKE_PREFIX_PATH.  One reason may be that no ROS setup.sh was
  sourced before.`

Same question was answerd here:
[https://stackoverflow.com/questions/64686050/yocto-sdk-ros1-melodic-meta-ros-build?answertab=votes#tab-top](https://stackoverflow.com/questions/64686050/yocto-sdk-ros1-melodic-meta-ros-build?answertab=votes#tab-top)

Something seems still broken.
